src/lib/template-instance.ts

```diff
@@ -41,20 +41,20 @@ export class TemplateInstance {
 
   update(values: unknown[]) {
     let i = 0;
-    for (const part of this._parts) {
+    for (const part of this.__parts) {
       if (part !== undefined) {
         part.setValue(values[i]);
       }
       i++;
     }
-    for (const part of this._parts) {
+    for (const part of this.__parts) {
       if (part !== undefined) {
         part.commit();
       }
     }
   }
 
-  _clone(): DocumentFragment {
+  clone(): DocumentFragment {
```

Comment:
I don't think we should rename this until 2.0.

Reply:
Undone.